### ISSUE TITLE
Questions about Flow-matching supervision

### ISSUE THREAD

**Issue by 1290833203** [CSU-JPG/Glance#3]
It's very meaningful work. But my abilities are limited, and I want to understand why the training objective of Flow Matching can be used as a distillation loss function. Why does using this objective allow for jump-step sampling generation after training for certain predefined time steps? Shouldn't this loss be modeling the instantaneous velocity? LoRA seems to train only at certain time steps, so why is it possible to perform jump-step sampling after training at these time steps? I would greatly appreciate it if you could provide me with some explanations or references.

<img width="295" height="320" alt="Image" src="https://github.com/user-attachments/assets/24d78674-05ed-400b-a8a5-4548454fadb2" />

**Comment by zhuobaidong** [comment]
Great question!

At the beginning we tried a simple MSE loss, but it didn’t work very well in practice. Switching to a Flow Matching–style loss helped a lot. Our intuition is that FM learns a velocity field instead of just matching states, which makes it naturally suitable for distillation.

Although velocity is defined in continuous time, we only train on a carefully selected set of timesteps and the jump-step sampling uses exactly the same schedule. So this is not arbitrary skipping.

We believe this works well with one data because LoRA mainly activate and reshape the latent capacity of the base model, rather than learning the flow field from scratch. In other words, the pretrained base model already encodes a strong prior of the diffusion/flow geometry, and the Flow Matching objective helps expose this structure in a controllable way.

For related works, you can reference "TimeStep Master"(https://arxiv.org/pdf/2503.07416).
